index.html

```diff
@@ -13,11 +13,51 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Hammersmith+One&family=Lato&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="navbar">
+      <div class="row">
+        <div class="column-full">
+          <h2 class="title">Code Journal</h2>
+        </div>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="form-div" data-view="entry-form">
+      <form action="index.html" class="form">
+        <div class="row">
+          <div class="column-full">
+            <div class="header">
+              <h1>New Entry</h1>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
+            <div class="container">
```

Comment:
Why is the container so deeply nested? All of the main content for the grid should be in a `container`, with `row` and `column` elements within that.